src/app/functions/server/refreshStoredScanResults.ts

```diff
@@ -23,16 +23,24 @@ export async function refreshStoredScanResults(profileId: number) {
     const remoteScans = (await listScans(profileId)).data;
     const localScans = await getAllScansForProfile(profileId);
 
-    const newScans = remoteScans.filter(
+    const newScan = remoteScans.filter(
       (remoteScan) =>
         !localScans.some(
           (localScan) => localScan.onerep_scan_id === remoteScan.id,
         ),
     );
 
+    newScan.forEach((scan) =>
```

Comment:
Do we expect `newScan` to ever contain more than one item? The renaming of the variable `newScans` -> `newScan` indicates we do not?

Reply:
Oh it might, sorry that was just at typo. I'll make it plural `newScans` just to make it self-documenting